src/main/java/org/rumbledb/runtime/primary/StringRuntimeIterator.java

```diff
@@ -20,7 +20,7 @@
 
 package org.rumbledb.runtime.primary;
 
-import org.apache.commons.lang3.StringEscapeUtils;
```

Comment:
I wanted to point out that these imports have been updated in ObjectItem ArrayItem and here. I wasn't sure if this was performed by you or the IDE. I doubt this change affects anything

Reply:
That's on purpose. It fixes a warning that has been there for a long type: the old package was deprecated.